js/contextMenus.js

```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
         CommonMenu.separatorMenuItem,
         CommonMenu.importBookmarksMenuItem()
       ]
-    }, {
-      label: locale.translation('bravery'),
-      submenu: [
-        CommonMenu.braveryGlobalMenuItem(),
-        CommonMenu.braverySiteMenuItem()
-      ]
     },
     CommonMenu.downloadsMenuItem(),
+    CommonMenu.findOnPageMenuItem(),
+    CommonMenu.printMenuItem(),
     CommonMenu.separatorMenuItem,
-    {
-      label: locale.translation('help'),
-      submenu: [
-        CommonMenu.aboutBraveMenuItem(),
-        CommonMenu.separatorMenuItem,
-        CommonMenu.checkForUpdateMenuItem(),
```

Comment:
Ditto I don't think we should get rid of about brave and check for updates.
CC @bradleyrichter